cpp/include/cudf/copying.hpp

```diff
@@ -34,13 +34,13 @@ namespace cudf {
  */
 
 /**
- * @brief How to treat out of bounds indices. 
+ * @brief Policy to apply to account for possible out-of-bound indices
  * 
  * `NULLIFY` means to nullify output values corresponding to out-of-bounds gather_map values. 
- * `IGNORE` means to ignore values in `gather_map` that are out of bounds.
+ * `DONT_CHECK` means to not to check whether the indices are out-of-bounds.
  */
 
-enum class out_of_bounds_policy : int8_t { NULLIFY, IGNORE };
+enum class out_of_bounds_policy : int8_t { NULLIFY, DONT_CHECK };
```

Comment:
Move the docs from above. 
```suggestion
enum class out_of_bounds_policy : int8_t { 
   NULLIFY, /// Output values corresponding to out-of-bounds indices are null
   DONT_CHECK /// No bounds checking is performed
};
```